.circleci/config.yml

```diff
@@ -147,12 +151,12 @@ jobs:
 build_matrix: &build_matrix
   matrix:
     parameters:
-      os: [ alpine, debian, macos, "win/default" ]
+      os: [ alpine, debian, linux, macos, "win/default" ]
```

Comment:
@robertgzr why are we running things on `alpine` and `debian` in the first place? Why not just `linux`?

Reply:
the idea was to have a similar matrix to what we have in starship for macos versions. I picked debian, because that's closely related to ubuntu that the executor uses and alpine, since it uses musl-libc

Reply:
@robertgzr we only use `linux` on starship for linux stuff and we don't have any usage of `alpine` or `debian` there

Reply:
@robertgzr Why are we building the postject on multiple Linux executors? Do we miss anything by just test the artifact built on a single executor on the test matrix?

Reply:
I have sent https://github.com/postmanlabs/postject/pull/26 to stop building and testing on `alpine` and `debian` because we are running things on the `linux` executor already.